src/core/openthread.cpp

```diff
@@ -66,9 +66,6 @@ static otDEFINE_ALIGNED_VAR(sInstanceRaw, sizeof(otInstance), uint64_t);
 otInstance *sInstance = NULL;
 #endif
 
-void OT_CDECL operator delete(void *, size_t) throw() { }
-void OT_CDECL operator delete(void *) throw() { }
```

Comment:
Sorry, I see that these are only definitions... anyway, we might need to guard these definitions just for Windows then, if it really needs them.